alpha-build-machinery/scripts/update-deps.sh

```diff
@@ -1,27 +1,23 @@
 #!/bin/bash -e
 
-readonly GLIDE_MINOR_VERSION="13"
-readonly REQUIRED_GLIDE_VERSION="0.$GLIDE_MINOR_VERSION"
+readonly GO_MINOR_VERSION=12
```

Comment:
13 as that's the first version with modules on by default, unless these commands start setting the GO111MODULE env var, right?

Reply:
maybe we can just drop the check entirely since go version is mandated globally and go mod has compatibility guarantees